NumLib/DOF/DOFTableUtil.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef NUMLIB_DOF_DOFTABLEUTIL_H
+#define NUMLIB_DOF_DOFTABLEUTIL_H
+
+#include "NumLib/DOF/LocalToGlobalIndexMap.h"
+
+namespace NumLib
+{
+//! Returns nodal indices for the item identified by \c id from the given
+//!  \c dof_table.
+std::vector<GlobalIndexType> getIndices(
+    std::size_t const id, NumLib::LocalToGlobalIndexMap const& dof_table);
+
+//! Returns the values for the given \c dof_indices from the vector \c x.
+std::vector<double> getLocalNodalDOFs(
```

Comment:
Would probably be the better choice. However, I didn't want to create a second file for that single function. Or do you have an existing file in mind?
May I keep the function here for the time being?